main.go

```diff
@@ -1140,22 +1224,20 @@ func writeDesired(file *os.File, posLat, posLng, lastAng, desDist float64) (floa
 	return posLat, posLng
 }
 
-func samplePosition(odometry movementsensor.MovementSensor, lat, lng []float64, isLat bool, data *os.File) ([]float64, []float64) {
+func samplePosition(ctx context.Context, odometry movementsensor.MovementSensor, lat, lng []float64, isLat bool, data *os.File, done func()) ([]float64, []float64) {
 	lastLat, lastLng := 0.0, 0.0
-	for {
-		linVel, err := odometry.LinearVelocity(context.Background(), nil)
-		if err != nil {
-			logger.Error(err)
-			return nil, nil
-		}
-		if linVel.Y != 0 {
-			break
+	ticker := time.NewTicker(100 * time.Millisecond)
```

Comment:
make the 100 a constant at the top of the code so its easy to modify.

Reply:
I actually don't think they need to be different I was just trying things out and never made them the same value, I'll change it